blocks/library/paragraph/test/__snapshots__/index.js.snap

```diff
@@ -3,31 +3,25 @@
 exports[`core/paragraph block edit matches snapshot 1`] = `
 <div>
    
-  <div>
-    <div
-      class="blocks-rich-text"
-    >
+  <div
+    class="blocks-rich-text"
+  >
+    <div>
       <div>
-        <div>
-          <div
-            class="components-autocomplete"
-          >
-            <p
-              aria-autocomplete="list"
-              aria-expanded="false"
-              aria-label="Add text or type / to add content"
-              aria-multiline="false"
-              class="wp-block-paragraph blocks-rich-text__tinymce"
-              contenteditable="true"
-              data-is-placeholder-visible="true"
-              role="textbox"
-            />
-            <p
-              class="blocks-rich-text__tinymce wp-block-paragraph"
-            >
-              Add text or type / to add content
-            </p>
-          </div>
+        <div
+          class="components-autocomplete"
+        >
+          <p
+            aria-autocomplete="list"
+            aria-expanded="false"
+            aria-label="Add text or type / to add content"
+            aria-multiline="false"
+            class="wp-block-paragraph blocks-rich-text__tinymce"
+            contenteditable="true"
+            data-is-placeholder-visible="false"
```

Comment:
This is very unexpected. It looks like context provider doesn't work properly with Enzyme, but I can't observe the same in the browser. Very annoying thing :(